src/c3chart/CommonND.js

```diff
@@ -16,7 +16,7 @@
         };
 
         this._config.data.onclick = function (d, element) {
-            context.click(context.rowToObj(context._data[d.index]), d.id);
+            context.click(context.rowToObj(context.data()[d.index]), d.id);
```

Comment:
need to add this -> https://github.com/hpcc-systems/Visualization/blob/master/src/chart/Column.js#L89